test/python/test_authn_api.py

```diff
@@ -58,25 +61,47 @@ def test_login(self):
         Gets the API key of a user given the username and password via HTTP Basic Authentication.
         """
         # Attempt to login
-        self.api.login(self.account)
+        self.api.login('authn', self.account)
 
         # Ensure we cannot login with a bad password
         self.config.password = "FakePassword123"
 
         with self.assertRaises(openapi_client.exceptions.ApiException):
-            self.api.login(self.account)
+            self.api.login('authn', self.account)
+
+    # We dont have this authenticator setup so we cant authenticate with it
+    @unittest.expectedFailure
+    def test_service_login_200(self):
```

Comment:
Should the spec env have an external authenticator setup before these changes/in this PR? I think expecting these to fail proves that we don't know if the endpoint as defined is functional yet.

Reply:
My worry was that It would take a while to figure out how to get an external authenticator working, and I didn't want finishing all the endpoints to be blocked by it. I will take a look and see if I can get something working